src/pids.c

```diff
@@ -96,6 +97,99 @@ char *utf8_encode(int encoding, char *buf, int len)
     return NULL;
 }
 
+void report(pids_t *st)
+{
+    int i;
+    char *country_code = NULL;
+    char *name = NULL;
```

Comment:
Since ```country_code``` and ```name``` should not be freed, consider marking them as ```const char *```.

Reply:
Done.